cmd/keymasterd/main.go

```diff
@@ -2378,9 +2380,11 @@ func main() {
 			tls.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,
 		},
 	}
+	logFilterHandler := NewLogFilerHandler(http.DefaultServeMux, publicLogs)
```

Comment:
No need to register another handler. Instead, use `logbuf.NewWithOptions` to create a logger without adding HTTP handlers to a `ServeMux`.

Reply:
There is a similar convenience function: `lib/log/serverlogger.NewWithOptions` that can be used.